### DISCUSSION TITLE
HEXACOPTER Servo support

### DISCUSSION THREAD

**Discussion by rizacelik** [iNavFlight/inav#9096] [Q&A] [answered]
I recompiled the INAV Matek11 firmware build for the HEXACOPTER (HEX X) 6 Motors and 2 servos.
6 When the motor is added, there is no servo output. When there are 5 motors, there is an output in the servos.
Doesn't INAV 6 offer motor and Servo support?

**Comment by mmosca** [comment, marked as answer]
It is probably a hardware limitation.

Motors and servos can't share the same timer and the mcu has a limited number of timers.

If 2 pins use the same timer they can only be servos or motors, not a mix of them.
